src/Analyzer.JsonRuleEngine/JsonRuleEngine.cs

```diff
@@ -49,6 +49,51 @@ public static JsonRuleEngine Create(string rawRuleDefinitions, BuildILineNumberR
             return new JsonRuleEngine(ParseRuleDefinitions(rawRuleDefinitions), jsonLineNumberResolverBuilder);
         }
 
+        /// <summary>
+        /// Modifies the rules to run based on values defined in the configurations file.
+        /// </summary>
+        /// <param name="configuration">The template configurations to filter and analyze.</param>
+        public void FilterRules(string configuration)
+        {
+            var result = new List<RuleDefinition>();
```

Comment:
I'd recommend renaming this variable. These aren't results (violations found in templates), but the rules themselves.